examples/scene/bar_plot.py

```diff
@@ -1,3 +1,14 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 2
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Bar plot with Axis 
+=====================
```

Comment:
The underline must be the same number of characters (width) as the title.

Reply:
Okay, fixed